### PR TITLE
OADP-2786: OADP Velero received EOF, stopping recv loop

### PR DESCRIPTION
**Pull request by anarnold97**
* [OADP-2768](https://issues.redhat.com/browse/OADP-2768)
* I have added to **Known Issues for OADP 1.1.4 release notes**. I know it is not a known issue, but think the reader might look for it as a **Known Issue**. Plus it allows me to provide a link to the explanation of why this is `expected behaviour` and not a `bug`.

### Version

* OCP 4.11 → branch/enterprise-4.11
* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15

* **OADP 1.1.4++**

#### OpenShift

* Enterprise 4.11 → branch/enterprise-4.11
* Enterprise 4.12 → branch/enterprise-4.12
* Enterprise 4.13 → branch/enterprise-4.13
* Enterprise 4.14 → branch/enterprise-4.14

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):

### Issue:

* [https://issues.redhat.com/browse/OADP-2768](https://issues.redhat.com/browse/OADP-2768) 

### Link to docs preview:

* [OADP 1.1.4 release notes](https://66242--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-release-notes#known-issues1.1.5_oadp-release-notes)
* [Velero plugins section](https://66242--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins#oadp-configuring-velero-plugins_oadp-features-plugins)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kalexand-rh** [comment]
I've opened the `enterprise-4.15` branch. Please review your PR and update the applicable versions to also cover 4.15 if you expect that the changes in this PR will be valid in that branch.

**Comment by PrasadJoshi12** [comment]
@anarnold97 are we going to add it under known issues section of every release ? I see the fix version for this bug is 1.3.0.

**Comment by anarnold97** [comment]
> @anarnold97 are we going to add it under known issues section of every release ? I see the fix version for this bug is 1.3.0.

@PrasadJoshi12 - I was adding it as a **known** issue in 1.1.4 and as a **resolved** issue in 1.3.0
What are your thoughts?

**Comment by anarnold97** [comment]
@PrasadJoshi12 - how about you consider approving this PR and I will highlight to Carmi, providing him with text to include for the 1.3.0 release notes he is currently working on?
Thanks

**Review by PrasadJoshi12 — APPROVED** [review]

**Comment by PrasadJoshi12** [comment]
> @PrasadJoshi12 - how about you consider approving this PR and I will highlight to Carmi, providing him with text to include for the 1.3.0 release notes he is currently working on? Thanks

Agree. LGTM

**Comment by anarnold97** [comment]
@PrasadJoshi12 - thanks

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by anarnold97** [comment]
> /label peer-review-done /remove-label peer-review-needed

@GroceryBoyJr - thanks and WOO-HOO!

File: modules/oadp-plugins-receiving-eof-message.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
```
Comment (stevsmit) [inline modules/oadp-plugins-receiving-eof-message.adoc:5]:
```suggestion
:_mod-docs-content-type: CONCEPT
```

File: modules/oadp-plugins-receiving-eof-message.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+[id="oadp-plugins-receiving-eof-message_{context}"]
+
+= Velero plugins returning "received EOF, stopping recv loop" message
+
+[NOTE]
+====
+Velero plugins are started as separate processes. Once the Velero operation has completed, either successfully or not, they exit. Therefore, if you see a `received EOF, stopping recv loop` messages in debug logs, it does not mean an error occurred, it means that a plugin operation has completed.
```
Comment (stevsmit) [inline modules/oadp-plugins-receiving-eof-message.adoc:12]: Offering up a slight suggestion here. Take it or leave it. At the very least, "once" needs swapped out: 

Use only to mean one time. Do not use as a conjunction to mean after or when.

See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o (once)

```suggestion
Velero plugins are started as separate processes. After the Velero operation has completed, either successfully or not, they exit. Receiving a `received EOF, stopping recv loop` message in the debug logs indicates that a plugin operation has completed. It does not mean that an error has occurred. 
```

File: modules/oadp-release-notes-1-1-4.adoc [hunk 3]
```diff
@@ -60,4 +60,9 @@ OADP backups might fail because a UID/GID range might have changed on the cluste
 
 A restoration might fail if ArgoCD is used during the process due to a label used by ArgoCD, `app.kubernetes.io/instance`. This label identifies which resources ArgoCD needs to manage, which can create a conflict with OADP's procedure for managing resources on restoration. To work around this issue, set `.spec.resourceTrackingMethod` on the ArgoCD YAML to `annotation+label` or `annotation`. If the issue continues to persist, then disable ArgoCD before beginning to restore, and enable it again when restoration is finished.
 
+
+.OADP Velero plugins returning "received EOF, stopping recv loop" message
+
+Velero plugins are started as separate processes. Once the Velero operation has completed, either successfully or not, they exit. Therefore, if you see a `received EOF, stopping recv loop` messages in debug logs, it does not mean an error occurred, it means that a plugin operation has completed. link:https://issues.redhat.com/browse/OADP-2176[OADP-2176]
```
Comment (stevsmit) [inline modules/oadp-release-notes-1-1-4.adoc:66]: Whatever you apply above, apply here. At the very least, once needs removed.

**Comment by stevsmit** [comment]
@anarnold97 3 nits. Also, be sure to squash your commits before re-adding the merge review needed label. There are currently 3 commits.

**Pull request closed without merging**